src/hooks/actions.tsx

```diff
@@ -238,7 +240,8 @@ export function useProposeAction({
           ]?.updateFromTransactionReceipt(receipt);
       })
       .catch(console.error);
-  }, [proposePriceTransaction, chainId, query]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [proposePriceTransaction]);
```

Comment:
removing query and chain to prevent infinite loop in hook. this would keep emitting notifications after tx completed, because this would then trigger a query update

Reply:
I should not have included the query and chain id here in the first place, so removing this is all good

Reply:
this wasnt you, i added this when i refactored